garnet/src/ConfirmPanelSample.js

```diff
@@ -0,0 +1,200 @@
+	}
+});
+
+var IconTitleTextPanel = kind({
+	name: 'confirmPanelWithIconNoScroll',
+	kind: ConfirmPanel,
+	events: {
+		onHidePanel: ''
+	},
+	handlers: {
+		onHide: 'hidePanel'
+	},
+	buttonComponents: [
+		{name: 'ok2', kind: IconButton, ontap: 'tapOK', classes: 'g-ok-image'}
+	],
+	components: [
+		{
+			kind: BlurPanelScroller,
+			icon: true,
+			iconSrc: '@../assets/ic_warning.svg',
+			title: true,
+			titleContent: 'test',
+			components: [
+				{content: 'All the labels follow guideline.'}
+			]
+		}
+	],
+	hidePanel: function(inSender, inEvent) {
+		this.doHidePanel({msg: inSender.name + ' is hidden'});
+	},
+	tapOK: function(inSender, inEvent) {
+		this.doHidePanel({msg: inSender.owner.name + ' is hidden by OK button'});
+	}
+});
+
+var ScrollIconTitleTextPanel = kind({
+	name: 'confirmPanelWithIconAndScroll',
+	kind: ConfirmPanel,
+	events: {
+		onHidePanel: ''
+	},
+	handlers: {
+		onHide: 'hidePanel',
+		onCancel: 'hidePanel',
+		onOK: 'tapOK'
+	},
+	components: [
+		{
+			kind: BlurPanelScroller,
+			icon: true,
+			iconSrc: '@../assets/ic_warning.svg',
+			title: true,
+			titleContent: 'ScrollIconTitleTextPanelTitle',
+			components: [
+				{content: 'Garnet is a UI library built for wearable devices and is based on Enyo. Garnet supports LG smart watch as well as the emulator provided. Browser-wise, Garnet supports the browser on LG smart watch, Firefox 16, Opera 15, Safari 3.1, Chrome 27 and their higher versions.et Quad-Core Processor, 1080p resolution screen, 13-megapixel rear'}
+			]
+		}
+	],
+	hidePanel: function(inSender, inEvent) {
+		this.doHidePanel({msg: inSender.name + ' is hidden'});
+	},
+	tapOK: function(inSender, inEvent) {
+		this.doHidePanel({msg: inSender.owner.name + ' is hidden by OK button'});
+	}
+});
+
+var ConfirmBasePanel = kind({
+	name: 'g.sample.ConfirmBasePanel',
+	kind: Panel,
+	events: {
+		onShow: ''
+	},
+	components: [
+		{name: 'button1', kind: Button, style: 'margin:' + ri.scale(20) + 'px ' + ri.scale(5) +'px ' + ri.scale(5) +'px; width:' + ri.scale(310) + 'px', ontap: 'showPanel', content: 'Only text'},
+		{name: 'button2', kind: Button, style: 'margin:' + ri.scale(5) + 'px; width:' + ri.scale(310) + 'px', ontap: 'showPanel', content: 'Scroll + Text'},
+		{name: 'button3', kind: Button, style: 'margin:' + ri.scale(5) + 'px; width:' + ri.scale(310) + 'px', ontap: 'showPanel', content: 'Icon + Title + Text'},
+		{name: 'button4', kind: Button, style: 'margin:' + ri.scale(5) + 'px; width:' + ri.scale(310) + 'px', ontap: 'showPanel', content: 'Scroll+Icon+Title+Text'}
+	],
+	showPanel: function(inSender, inEvent) {
+		var name = inSender.name;
+
+		if (name == 'button1' || name == 'button2' || name == 'button3' || name == 'button4') {
+			this.doShow({panelType: name, panelName: name});
+		}
+	}
+});
+
+module.exports = kind({
+	name: 'g.sample.ConfirmPanelSample',
+	horizontal: 'hidden',
+	classes: 'enyo-unselectable enyo-fit garnet g-sample',
+	kind: Scroller,
+	components: [
```

Comment:
Property orders:
kind > horizontal > classes > components
Please check kind-template.js in Garnet library.